pkg/kev/config/extensions_test.go

```diff
@@ -87,10 +79,14 @@ var _ = Describe("Extentions", func() {
 					}
 				})
 
-				It("returns an error", func() {
-					parsedCfg, err = config.K8SCfgFromMap(extensions)
-					Expect(err).To(HaveOccurred())
-					Expect(err.Error()).To(ContainSubstring("Workload.Replicas is required"))
+				JustBeforeEach(func() {
+					svc.Extensions = extensions
+				})
+
+				It("returns defaults", func() {
+					parsedCfg, err = config.K8SCfgFromCompose(&svc)
```

Comment:
A minor comment - the first lines here could be moved up to `JustBeforeEach` block and this `It` could just do the assertion on the `err` and `parsedCfg` values.

Reply:
fixed